projects/storefrontlib/src/shared/services/trigger/services/outlet-render-strategy.service.ts

```diff
@@ -31,11 +43,34 @@ export class OutletRenderStrategy extends RenderStrategy {
         template,
         config.position ? config.position : OutletPosition.BEFORE
       );
-      this.renderedCallers.push(caller);
+      this.outletRendererService.render(config.outlet);
+
+      const renderedCaller = vcr?.element
+        ? { caller, element: vcr.element }
+        : { caller };
+      this.renderedCallers.push(renderedCaller);
     }
   }
 
-  public strategy(config: TriggerOutletMapping) {
+  public isStrategyForConfiguration(config: TriggerOutletMapping) {
     return Boolean(config.outlet);
   }
+
+  public removeRendered(
```

Comment:
just `remove`?

Reply:
btw I tend to use public as the default access modifier, but it's fine. Might be my personal habit.

Reply:
Might be personal ;)